### DISCUSSION TITLE
feedback on 100daysofgatsby challenge 3 - requires $

### DISCUSSION THREAD

**Discussion by Kevpedia** [gatsbyjs/gatsby#29626] [Help]
@hashimwarren - RE [challenge 3](https://www.gatsbyjs.com/blog/100days-challenge-3), not sure if you realized, but adding the WPGraphQL plugin in WordPress requires a paid plan... I'm not going to pay money just to stand up this site, and many participants may not even be _able_ to do so.

<img width="774" alt="image" src="https://user-images.githubusercontent.com/16330251/108602374-98612700-736f-11eb-9039-2611c0679323.png">

**Comment by hashimwarren** [comment]
Thanks for that feedback on that barrier. I'll add a note about getting free/cheap WordPress hosting from Amazon Lightsail 👍
